packages/ai/core/generate-text/stream-text.test.ts

```diff
@@ -3429,19 +3429,21 @@ describe('streamText', () => {
               }
 
               // assuming test arg structure:
-              if (chunk.type === 'tool-call') {
+              if (chunk.type === 'tool-call' && chunk.args) {
                 chunk.args = {
                   ...chunk.args,
-                  value: chunk.args.value.toUpperCase(),
+                  value: (chunk.args as any).value.toUpperCase(),
```

Comment:
why is this now needed? indicates something might be wrong if the type inference